scenario/parse.go

```diff
@@ -89,8 +89,9 @@ func (s *Scenario) UnmarshalYAML(node *yaml.Node) error {
 			return gdterrors.ExpectedScalarAt(keyNode)
 		}
 		key := keyNode.Value
-		if key == "tests" {
-			valNode := node.Content[i+1]
+		valNode := node.Content[i+1]
```

Comment:
Still need to dig deeper in the yaml library to understand why we do `i += 2` and not just a `i++`.
But overall looks good!
/lgtm

Reply:
@a-hilaly yeah, it's weird... see the comment above:

```
	// maps/structs are stored in a top-level Node.Content field which is a
	// concatenated slice of Node pointers in pairs of key/values.
```